site/SiteFooter.tsx

```diff
@@ -14,267 +14,173 @@ interface SiteFooterProps {
-                                            height={127}
-                                        />
-                                    </picture>
-                                </a>
-                            </p>
-                        </div>
+        <footer className="site-footer grid grid-cols-12-full-width">
+            <div className="footer-left span-cols-5 span-sm-cols-12 col-start-2 col-sm-start-2">
+                <p className="body-3-medium">
+                    Our World in Data is a project of{" "}
+                    <a href="https://global-change-data-lab.org/">
+                        Global Change Data Lab
+                    </a>
+                    , a nonprofit based in the UK (Reg. Charity No. 1186433).
+                    Our charts, articles, and data are licensed under{" "}
+                    <a href="https://creativecommons.org/licenses/by/4.0/">
+                        CC BY
+                    </a>
+                    , unless stated otherwise. Tools and software we develop are
+                    open source under the{" "}
+                    <a href="https://github.com/owid/owid-grapher/blob/master/LICENSE.md">
+                        MIT license
+                    </a>
+                    . Third-party materials, including some charts and data, are
+                    subject to third-party licenses. See our{" "}
+                    <a href="/faqs">FAQs</a> for more details.
+                </p>
+                <div className="affiliates">
+                    <div className="oxford-logos">
+                        <a href="https://www.oxfordmartin.ox.ac.uk/global-development">
+                            <img
+                                src={`${props.baseUrl}/oms-logo.svg`}
+                                alt="Oxford Martin School logo"
+                                loading="lazy"
+                                width="58"
+                                height="62"
+                            />
+                        </a>
+                        <a href="https://www.ox.ac.uk">
+                            <img
+                                src={`${props.baseUrl}/oxford-logo.svg`}
+                                alt="University of Oxford logo"
+                                loading="lazy"
+                                width="58"
+                                height="62"
+                            />
+                        </a>
+                        <a href="https://global-change-data-lab.org">
+                            <img
+                                src={`${props.baseUrl}/gcdl-logo.svg`}
+                                alt="Global Change Data Lab logo"
+                                loading="lazy"
+                                width="106"
+                                height="62"
+                            />
+                        </a>
                     </div>
+                    <a href="https://www.ycombinator.com">
```

Comment:
nit: the click target for this link is super wide in a large viewport, double or triple the width of the actual logo.